client/app/lib/components/sidebarmachineslistitem/leavesharedmachinewidget.coffee

```diff
@@ -11,42 +11,57 @@ module.exports = class LeaveSharedMachineWidget extends React.Component
   @defaultProps =
     className : 'Approved'
 
+  constructor: (props) ->
+
+    super
+
+    @state = {
+      isModalActive : true
+    }
 
   onLeaveClicked: ->
 
+    console.log('@state ', @state)
+    @setState {isModalActive: false}
```

Comment:
here is the [coffee-styleguide](https://github.com/koding/styleguide-coffeescript) 
`obj = { foo: bar, baz: qux }`

Apply it for all other wrong usage.